omnibus/files/private-chef-cookbooks/private-chef/recipes/fix_permissions.rb

```diff
@@ -0,0 +1,11 @@
+LIB_PATH="/opt/opscode/embedded/lib"
+# The GEM_PATH should work since we allow only one version of ruby to be installed.
+GEM_PATH="#{LIB_PATH}/ruby/gems/*/gems"
+
+execute "find #{GEM_PATH} -executable -exec chmod 755 {} \\;" do
```

Comment:
From some very brief research, it looks like the -executable flag may only appear in findutils 4.4. It looks like Centos 5 ships 4.2.27 or such, and we still are building Centos 5 images for chef-server. Perhaps ```find . -perm /u=x,g=x,o=x```? (Actually, just u=x might be fine here)

Reply:
nice catch! and we do support el-5, got the tests to fail due to this error.